tests/v2/validation/auth/userretention/userretention_delete_user_test.go

```diff
@@ -0,0 +1,252 @@
+	userReLogin, err := auth.GetUserAfterLogin(ur.client, *newStdUser)
+	require.NoError(ur.T(), err)
+
+	logrus.Info("Updating user attributes")
+	userAttributes, err := ur.client.WranglerContext.Mgmt.UserAttribute().Get(newStdUser.ID, v1.GetOptions{})
+	require.NoError(ur.T(), err)
+	userAttributes.DeleteAfter = &v1.Duration{Duration: time.Second * 10}
+	_, err = ur.client.WranglerContext.Mgmt.UserAttribute().Update(userAttributes)
+	require.NoError(ur.T(), err)
+
+	logrus.Info("Verifying user status")
+	userReLogin, err = ur.client.WranglerContext.Mgmt.User().Get(userReLogin.Name, v1.GetOptions{})
+	assert.Equal(ur.T(), isActive, *(userReLogin).Enabled)
+	require.NoError(ur.T(), err)
+
+	logrus.Info("Polling user status to disable")
+	pollUserStatus(ur.client, newStdUser.ID, isInActive)
+
+	logrus.Info("Attempting login after wait period")
+	_, err = auth.GetUserAfterLogin(ur.client, *newStdUser)
+	assert.ErrorContains(ur.T(), err, "401 Unauthorized")
+
+	logrus.Info("Checking bindings after user deletion")
+	bindingsAfter, err := getBindings(ur.client, newStdUser.ID)
+	require.NoError(ur.T(), err)
+	assert.Empty(ur.T(), bindingsAfter["RoleBindings"], "Expected no RoleBindings after user deletion")
+	assert.Equal(ur.T(), 0, len(bindingsAfter["RoleBindings"].([]rbac.RoleBinding)), "RoleBindings slice should be empty after user deletion")
+
+	logrus.Info("Checking global role bindings after user deletion")
+	globalRoleBindingsAfter, err := ur.client.Management.GlobalRoleBinding.List(&types.ListOpts{
+		Filters: map[string]interface{}{
+			"userId": newStdUser.ID,
+		},
+	})
+	require.NoError(ur.T(), err)
+	assert.Empty(ur.T(), globalRoleBindingsAfter.Data, "Expected no GlobalRoleBindings after user deletion")
+}
+
+func (ur *URDeleteTestSuite) TestUserIsNotDeletedWithBlankSettings() {
+	logrus.Info("Starting TestUserIsNotDeletedWithBlankSettings")
+	logrus.Info("Setting up user retention settings with blank values")
+	setupUserRetentionSettings(ur.client, "", "400h", "*/1 * * * *", "false")
+
+	logrus.Info("Creating new standard user")
+	newUser, err := users.CreateUserWithRole(ur.client, users.UserConfig(), "user")
+	require.NoError(ur.T(), err)
+
+	logrus.Info("Logging in with new user")
+	userReLogin, err := auth.GetUserAfterLogin(ur.client, *newUser)
+	require.NoError(ur.T(), err)
+
+	logrus.Info("Updating user attributes")
+	userAttributes, err := ur.client.WranglerContext.Mgmt.UserAttribute().Get(newUser.ID, v1.GetOptions{})
+	require.NoError(ur.T(), err)
+	userAttributes.DeleteAfter = nil
+	_, err = ur.client.WranglerContext.Mgmt.UserAttribute().Update(userAttributes)
+	require.NoError(ur.T(), err)
+
+	logrus.Info("Verifying user status")
+	userReLogin, err = ur.client.WranglerContext.Mgmt.User().Get(userReLogin.Name, v1.GetOptions{})
+	assert.Equal(ur.T(), isActive, *(userReLogin).Enabled)
+	require.NoError(ur.T(), err)
+
+	logrus.Info("Waiting for default duration")
+	time.Sleep(defaultWaitDuration)
```

Comment:
Didn't we discuss about this to make use of polling? poll as an admin to get the user attributes or User resources... then we can avoid using sleep.